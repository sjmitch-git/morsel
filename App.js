import React from "react";
import Layout from "@/structure/Layout";
import HomeScreen from "@/screens/HomeScreen";

const App = () => {
  return (
    <Layout>
      <HomeScreen />
    </Layout>
  );
};

export default App;
