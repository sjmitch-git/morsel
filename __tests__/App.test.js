import "react-native";
import renderer from "react-test-renderer";

import App from "../App";

it("renders correctly", () => {
  renderer.create(<App />);
});

test("renders App component correctly", () => {
  const component = renderer.create(<App />);
  const instance = component.root;
  const text = instance.findByProps({ children: "Home Screen!" });
  expect(text).toBeTruthy();
});

test("renders App component correctly", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
