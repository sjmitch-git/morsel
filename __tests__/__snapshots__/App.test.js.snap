// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#ecf0f1",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "grey",
        "height": "15%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Text>
      Header Component
    </Text>
  </View>
  <View
    style={
      {
        "flex": 1,
        "padding": 15,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          {
            "fontSize": 20,
            "fontWeight": "bold",
          }
        }
      >
        Home Screen!
      </Text>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          [
            {
              "borderRadius": 5,
              "padding": 10,
            },
            {
              "borderRadius": 5,
              "padding": 10,
            },
            {
              "backgroundColor": "#3498db",
            },
            null,
          ]
        }
      >
        <Text
          style={
            {
              "color": "white",
              "fontSize": 16,
            }
          }
        >
          Send my Location
        </Text>
      </View>
    </View>
  </View>
</View>
`;
